examples/user_interaction/client/src/App.js

```diff
@@ -23,47 +23,49 @@ class App extends React.Component {
     this.state = { pi: { time: 0, value: "unknown" }, prime_numbers: { time: 0, value: [] } };
     this.handleClick = this.handleClick.bind(this);
     this.runSecondTask = this.runSecondTask.bind(this);
-    this.http = require('http');
```

Comment:
Now that you aren't using `http` here, can you remove the `axios` dependency in the `package.json` file?

Reply:
They were not present in the dependencies. However, the server had axios and I removed it.